src/k8s/cmd/k8s/k8s_join_cluster.go

```diff
@@ -47,13 +50,23 @@ var (
 				return fmt.Errorf("failed to create cluster client: %w", err)
 			}
 
-			err = client.JoinNode(cmd.Context(), joinNodeCmdOpts.name, joinNodeCmdOpts.address, token)
+			if c.IsBootstrapped(cmd.Context()) {
+				return fmt.Errorf("A k8s cluster is already running on this node.")
+			}
+			const minTimeout = 3
```

Comment:
nit, and use `%v` instead of `%ds` below
```suggestion
			const minTimeout = 3 * time.Second
```

Reply:
huh, weird - I thought I have already changed that - thanks!